### DISCUSSION TITLE
sodium.randombytes_buf is not a function

### DISCUSSION THREAD

**Discussion by vdt000** [ecadlabs/taquito#998] [Q&A] [answered]
Hi,
I got an email from airgap, says that: we thank you for your bug message, we deploy new version of wallet-min.js.
And new version working, no "sodium lib bug." Also "\node_modules@airgap\beacon-sdk" has new version 10. BUT still :
"TypeError: sodium.randombytes_buf is not a function
at Object. (\node_modules@airgap\beacon-sdk\dist\cjs\utils\generate-uuid.js:72:34)"
BUG is there.
We use this function:
const wallet = new BeaconWallet({ name: "Beacon Docs Taquito" });
and bug comes.
We must send tez money to our users via nodejs wallet library.
URGENT...

**Comment by roxaneletourneau** [comment, marked as answer]
Related to: https://github.com/ecadlabs/taquito/issues/999
